### PR TITLE
PLNSRVCE-1381: prototype adding cluster-wide results viewing to some pipeline-service team members

### PR DESCRIPTION
**Pull request by gabemontero**
what I think an initial permission bump in staging only would look like so either @sayan-biswas or myself could start prototyping direct viewing of logs / records of results in staging only

Of course let me know @sayan-biswas if I misremembered the resources / verbs needed

@redhat-appstudio/pipeline-service FYI

### CODE REVIEW

**Comment by gabemontero** [comment]
bump @sayan-biswas @Roming22 can I get a github approval so this can merge via the bot vs. me hitting the merge button

fyi since I am not updating the development overlay (just adding staging only RBAC) the need to run e2e's is excluded :-)

**Review by Roming22 — APPROVED** [review]

**Comment by sayan-biswas** [comment]
@gabemontero I might have missed this notification, apologies. 
We do need to add `log` under resources. I'll open another PR for this.

https://github.com/tektoncd/results/blob/main/config/base/default-clusterroles.yaml

**Pull request merged**
